docs/sources/setup-grafana/installation/kubernetes/index.md

```diff
@@ -9,114 +9,645 @@ weight: 500
+   ; # Either "debug", "info", "warn", "error", "critical", default is "info"
+   ; # we change from info to debug level
+   level = debug
+   ```
+
+   This example adds the portion of the log section from the configuration file. You can refer to the [Configure Grafana](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) documentation to view all the default configuration settings.
+
+1. To add the configuration file into the Kubernetes cluster via the ConfigMap object, run the following command:
+
+   ```bash
+   kubectl create configmap ge-config --from-file=/path/to/file/grafana.ini --namespace=my-grafana
+   ```
+
+1. To verify the ConfigMap object creation, run the following command:
+
+   ```bash
+   kubectl get configmap --namespace=my-grafana
+   ```
+
+1. Open the `grafana.yaml` file and In the Deployment section, provide the mount path to the custom configuration (`/etc/grafana`) and reference the newly created ConfigMap for it.
+
+   ```bash
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     labels:
+       app: grafana
+     name: grafana
+   # the rest of the code remains the same.
+   ...
+   ....
+   ...
+               requests:
+               cpu: 250m
+               memory: 750Mi
+           volumeMounts:
+             - mountPath: /var/lib/grafana
+               name: grafana-pv
+              # This is to mount the volume for the custom configuration
+             - mountPath: /etc/grafana
+               name: ge-config
+       volumes:
+         - name: grafana-pv
+           persistentVolumeClaim:
+             claimName: grafana-pvc
+          # This is to provide the reference to the ConfigMap for the volume
+         - name: ge-config
+           configMap:
+             name: ge-config
+   ```
+
+1. Deploy the manifest using the following kubectl apply command:
+
+   ```bash
+   kubectl apply -f grafana.yaml --namespace=my-grafana
+   ```
+
+1. To verify the status, run the following commands:
+
+   ```bash
+   # first check the rollout status
+   kubectl rollout status deployment grafana --namespace=my-grafana
+
+   # then check the deployment and configMap information
+   kubectl get all --namespace=my-grafana
+   ```
+
+1. To verify it, access the Grafana UI in the browser using the provided IP:Port
+
+   The Grafana sign-in page appears.
+
+1. To sign in to Grafana, enter `admin` for both the username and password.
+
+1. Navigate to **Server Admin > Settings** and then search for log.
+
+   You should see the level to debug mode.
+
+### Using the --dry-run command
+
+You can use the Kubernetes `--dry-run` command to send requests to modifying endpoints and determine if the request would have succeeded.
+
+Performing a dry run can be useful for catching errors or unintended consequences before they occur. For more information, refer to [Kubernetes Dry-run](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/576-dry-run/README.md).
+
+Example:
```

Comment:
Should this be a heading? (H4 maybe?) or a sentence explaining the example?

Reply:
@chri2547 can you please suggest here?

Reply:
@usmangt , we could add a sentence here that says something like "The following example shows..." - and then I need you to fill in what the example shows.

Reply:
@chri2547 just pushed the changes. please check